hips/tiles/tests/test_tile.py

```diff
@@ -4,45 +4,103 @@
 from numpy.testing import assert_allclose
 from numpy.testing import assert_equal
 from ..tile import HipsTile, HipsTileMeta
-
+from ...utils.testing import get_hips_extra_file, requires_hips_extra
 
 class TestHipsTile:
+    def assert_fits(self, tile1, tile2):
```

Comment:
I don't think it makes sense to split out these `assert_*` methods.
You call each of them twice: once from the read/write test, and once from the fetch test.

You can just copy these asserts at the end of the read / write tests, and then by simply having `tile_read == tile_fetched` at the end of the fetch tests, you'll have tested exactly the same thing, without the extra complexity of these three separate methods.